resources/perl-libs/README.md

```diff
@@ -0,0 +1,29 @@
+# SQE_DB_API
```

Comment:
Are these accidentally included? I thought that they were going to stay in the other repo and we would clone them in here.

Reply:
Good catch!  There should be no resources/perl-libs folder.  It is in .gitignore, so I have made some mistake somewhere.  I will try to fix it.